frontend/src/components/form/base/EColorPicker.vue

```diff
@@ -34,7 +34,7 @@ export default {
   name: 'ColorPicker',
   components: { BasePicker },
   props: {
-    value: { type: String, required: true }
+    value: { type: Object, required: true }
```

Comment:
Revert, change does not belong in this PR
```suggestion
    value: { type: String, required: true }
```